tee-worker/cli/src/trusted_operation.rs

```diff
@@ -74,17 +75,10 @@ pub(crate) fn perform_trusted_operation<T: Decode + Debug>(
 pub(crate) fn perform_direct_operation<T: Decode + Debug>(
 	cli: &Cli,
 	trusted_args: &TrustedCli,
-	top: &TrustedOperation<TrustedCallSigned, Getter>,
+	top: &TrustedVCRequestSigned,
```

Comment:
So `perform_direct_operation` will be limited only for VC request operations? If yes, `perform_direct_operation` can be renamed to `perform_direct_vc_request_operation`. 
I just think the scenario would be different. There might be other operation in the future which also needs direct operation. How do you think so?

Reply:
I would say for now I will change it to `perform_direct_vc_request_operation` and create a separate issue to move this operation to `TrustedOperation`.